docker/docker-cluster/metrictank.ini

```diff
@@ -148,6 +148,8 @@ log-min-dur = 5min
 time-zone = local
 # maximum number of concurrent threads for fetching data on the local node. Each thread handles a single series.
 get-targets-concurrency = 20
+# what the default limit for tagdb query results should be
```

Comment:
"what the... should be" is redundant.
and if this setting is only a default, we should document what may override it

Reply:
https://github.com/grafana/metrictank/pull/779/commits/9d217cd83c383b8046976e88be2a0f6c43946d2f